packages/vite-plugin-cloudflare/src/miniflare-options.ts

```diff
@@ -584,7 +585,7 @@ export function getPreviewMiniflareOptions(
 		];
 	});
 
-	const logger = new ViteMiniflareLogger(resolvedViteConfig);
+	const logger = new ViteMiniflareLogger(resolvedViteConfig, true);
```

Comment:
You get better self documenting if you put options like this into an object.

```suggestion
	const logger = new ViteMiniflareLogger(resolvedViteConfig, { isPreview: true });
```